ngraph/frontend/tensorflow/src/op/elu.cpp

```diff
@@ -0,0 +1,23 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <default_opset.h>
+
+#include <op_table.hpp>
+#include <tensorflow_frontend/node_context.hpp>
+
+using namespace std;
+using namespace ngraph;
+using namespace ngraph::frontend::tensorflow::detail;
+
+namespace tensorflow {
+namespace ngraph_bridge {
```

Comment:
can we consider using ngraph::frontend::tf namespace for loaders/transformations? Now I am a bit lost due to a number of namespaces:)

Reply:
need to replace it everywhere, not sure it's a part of this task

Reply:
will be done in the separate PR